src/scene/vcLiveFeed.cpp

```diff
@@ -306,6 +305,8 @@ vcLiveFeed::vcLiveFeed(vdkProject *pProject, vdkProjectNode *pNode, vcState *pPr
   m_oldestFeedUpdate = m_newestFeedUpdate;
   m_fetchNow = true;
 
+  m_labelLODModifier = 1.0;
```

Comment:
This was moved out of the initialiser list?

Reply:
one of the build machines had a problem with it

Reply:
There is a rule in C++ that the list here has to be in the same order as they are defined in the header; you probably needed to move it above the mutex.

Reply:
Most compilers don't care though